RawWaterManagement/SluiceGate/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,142 @@
+    "value": {
+      "streetAddress": "Its for four. Order pass rich wide material deal. Group dog property painting.",
+      "addressLocality": "Culture condition involve. North us really door newspaper. Door any again.",
+      "addressRegion": "Audience watch much measure. Admit we across know fill since.",
+      "addressCountry": "Her million artist management almost.",
+      "postalCode": "Land do avoid watch next economic today. Perhaps practice money. Measure rather no identify happen plan.",
+      "postOfficeBoxNumber": "Today from indeed measure. Father since child. This information difference program policy full. Foreign contain live.",
+      "areaServed": "Away defense politics. Parent scientist source town federal crime. All short drive while south skin."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Court item business when idea thousand anything. Something know option as. Begin operation show place attack lead itself."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:EGUP:16321644",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2003-06-19T06:37:36Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2019-02-20T21:54:19Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Else standard campaign never."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Military successful report wear necessary let research. Decade choice cell."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Cost tell site concern cost. West whom hope feel nothing candidate political. Government knowledge dog inside."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Too commercial a step. Bill another choice next suffer. Generation bank spend happen police."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Than produce spend executive talk fall relationship. Place fire require each up yet participant."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:RVGI:73108415",
+      "urn:ngsi-ld:dataModel:items:BKDJ:84525829"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:UREZ:52029940",
+      "urn:ngsi-ld:dataModel:items:ZMTO:27892388"
+    ]
+  },
+  "type": "SluiceGate",
+  "tag": {
+    "type": "Property",
+    "value": "Leg great small every wind the really. Its safe bad condition break nation value participant."
+  },
+  "gateType": {
+    "type": "Property",
+    "value": "Sluice Gate"
+  },
+  "gateWidth": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "gateOpening": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "gateBottomElevation": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "gateDischargeCoefficient": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "orificeDischargeCoefficient": {
```

Comment:
Use of unitcode is encouraged to ensure consistency of collected data. Otherwise, no one knows if e.g. a temperature is expressed in °C or K 
The specification recommends to take it from UN CEFACT (https://unece.org/DAM/cefact/recommendations/rec20/rec20.zip)